modules/transactions/src/main/java/org/apache/ignite/internal/tx/storage/state/rocksdb/TxStateRocksDbStorage.java

```diff
@@ -47,26 +47,16 @@
 import org.apache.ignite.internal.util.IgniteSpinBusyLock;
 import org.jetbrains.annotations.Nullable;
 import org.rocksdb.ReadOptions;
-import org.rocksdb.RocksDB;
 import org.rocksdb.RocksDBException;
 import org.rocksdb.RocksIterator;
 import org.rocksdb.Slice;
 import org.rocksdb.WriteBatch;
-import org.rocksdb.WriteOptions;
 
 /**
  * Tx state storage implementation based on RocksDB.
  */
 public class TxStateRocksDbStorage implements TxStateStorage {
-    /** RocksDB database. */
-    private final RocksDB db;
-
-    /** Write options. */
-    private final WriteOptions writeOptions;
-
-    /** Read options for regular reads. */
-    private final ReadOptions readOptions;
-
+    private static final int PREFIX_SIZE_BYTES = Integer.BYTES + Short.BYTES;
```

Comment:
Can you indicate what will be stored in the prefix? for example, table and partition ID.